### DISCUSSION TITLE
Welcome to linuxcnc-ethercat Discussions!

### DISCUSSION THREAD

**Discussion by scottlaird** [linuxcnc-ethercat/linuxcnc-ethercat#9] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by EhsanRasouli1371** [comment]
What should we do if we want to use a servo motor that is not on the list of supported devices?

**Reply by EhsanRasouli1371** [reply]
Can you create a manual for how to use a new device?
